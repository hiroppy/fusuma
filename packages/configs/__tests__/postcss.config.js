describe('postcss', () => {
  afterEach(() => {
    process.env.NODE_ENV = 'test';

    jest.resetModules();
  });

  it('should match settings when NODE_ENV is development', () => {
    process.env.NODE_ENV = 'development';

    const postcss = require('../src/postcss.config')();

    expect(postcss).toMatchSnapshot();
  });

  it('should match settings when NODE_ENV is production', () => {
    process.env.NODE_ENV = 'production';

    const postcss = require('../src/postcss.config')();

    expect(postcss).toMatchSnapshot();
  });
});
