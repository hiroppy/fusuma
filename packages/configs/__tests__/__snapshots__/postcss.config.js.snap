// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`postcss should match settings when NODE_ENV is development 1`] = `
Object {
  "plugins": Array [
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
  ],
}
`;

exports[`postcss should match settings when NODE_ENV is production 1`] = `
Object {
  "plugins": Array [
    [Function],
    [Function],
    [Function],
    [Function],
  ],
}
`;
